src/components/Content/Content.tsx

```diff
@@ -54,43 +55,51 @@ class Content extends React.Component<any, any> {
       styles: stylesProp = {},
       id,
       emptyMessage,
-      effects: { saveChanges },
-      state: { item },
+      effects: { saveChanges, setItem },
+      state: { item, valid },
+      type,
+      history,
     } = this.props;
 
+    const { creating, editing, saving } = this.state;
     return (
       <div className={`content ${css(styles.container, stylesProp)}`}>
         <ControlContainer style={styles.controls}>
-          <div>
-            {!this.state.editing && (
-              <Button
-                basic
-                color="green"
-                onClick={() => {}}
-                size="tiny"
-                style={{ fontWeight: 'bold' }}
-              >
-                Create
-              </Button>
-            )}
-            {!this.state.editing &&
-              id && (
+          {!editing &&
+            !creating && (
+              <div>
                 <Button
-                  color="blue"
-                  onClick={() => this.setState({ editing: true })}
+                  basic
+                  color="green"
+                  onClick={async () => {
+                    await setItem(null, type);
+                    this.setState({ creating: true, editing: false });
+                  }}
                   size="tiny"
-                  style={{ fontWeight: 'normal' }}
+                  style={{ fontWeight: 'bold' }}
                 >
-                  Edit
+                  Create
```

Comment:
is this git diff being dumb?

Reply:
Yeah I changed the indentation so thats affecting the diff.